examples/settings.gradle

```diff
@@ -0,0 +1,2 @@
+include ':android-simple'
+include ':android-activity-graphs'
```

Comment:
I wasn't able to build the third with Android Studio. Anyone who is able to do so, can share his knowledge